### DISCUSSION TITLE
icons are not displayed.

### DISCUSSION THREAD

**Discussion by fredmac48** [LiveHelperChat/livehelperchat#1930] [Q&A]
In my installation the icons are not shown, I have already done all the steps that I have found but I can't get it to work.
I have installed in an IIS and php8; I get the following error in the console, and I don't know where to look for it to correct it.

Additionally, for some time now, a message appears at the top that says:

You have weak internet connection, or the server has problems. Try to refresh the page. Error code [404]

and then it disappears

I would appreciate any help.

sorry for my English

GET https://chatsoporte.ivolucion.com/design/defaulttheme/fonts/MaterialSymbolsRounded2.woff2 net::ERR_ABORTED 404 (Not Found)
53daa665fc5c26fd753c137cf4fa07b7.js:110          
GET https://chatsoporte.ivolucion.com/index.php/site_admin/chat/syncadmininterface/(limita)/10/(limitu)/10/(limitp)/10/(limito)/10/(limitc)/10/(limitd)/10/(limits)/10/(limitmc)/10/(limitb)/10/(limitgc)/10/(w)/1/9/4/3/5/8/6/(lda)/1680636685 404 (Not Found)

![image](https://user-images.githubusercontent.com/28351716/229900436-561b3dd1-8218-4c56-a375-9a2d5d9ad99d.png)

![image](https://user-images.githubusercontent.com/28351716/229900917-bbf9d76d-4822-4b71-b827-8306af679403.png)

 Thks

**Comment by remdex** [comment]
Looks like you did not uploaded all files compare with master branch and you will see that file existing in that branch

**Reply by fredmac48** [reply]
Thanks for the help, but it didn't work, I already copied the entire master branch and it's still the same

any other suggestions

![image](https://user-images.githubusercontent.com/28351716/229930905-e284ae4b-d06c-4285-b59a-6db917bcd68a.png)


![image](https://user-images.githubusercontent.com/28351716/229933091-bd204a57-4236-447a-8c85-314aa2a35d0f.png)

**Reply by remdex** [reply]
There is some threqd where you have to adjust IIS maximum path arguments.
Also checkanuslly that font file exists

Also you can try to clear cache from back office.

**Reply by remdex** [reply]
Might be relevant
https://github.com/LiveHelperChat/livehelperchat/issues/1479#issuecomment-641239123

**Reply by fredmac48** [reply]
> Podría ser relevante [#1479 (comentario)](https://github.com/LiveHelperChat/livehelperchat/issues/1479#issuecomment-641239123)

Thank you, you were able to solve the 404 error message, but I couldn't solve the one with the icons.
:'(

I don't know if it has something to do with the .htaccess file

![image](https://user-images.githubusercontent.com/28351716/230107588-833408b0-41bc-454b-93cf-15c0c204918a.png)
